tasks/migrations/0001_initial.py

```diff
@@ -0,0 +1,48 @@
+# Generated by Django 3.2.9 on 2021-11-17 20:10
+
+from django.db import migrations, models
+import django.db.models.deletion
+import enumchoicefield.fields
+import tasks.models
+
+
+class Migration(migrations.Migration):
+    initial = True
+
+    dependencies = [
+        ('users', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Task',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('title', models.CharField(max_length=200)),
+                ('created_date', models.DateTimeField(auto_now_add=True)),
+                ('priority', enumchoicefield.fields.EnumChoiceField(default=tasks.models.Priority(1),
+                                                                    enum_class=tasks.models.Priority, max_length=1)),
+                ('status',
+                 enumchoicefield.fields.EnumChoiceField(default=tasks.models.Status(1), enum_class=tasks.models.Status,
```

Comment:
```suggestion
                 enumchoicefield.fields.EnumChoiceField(default=tasks.models.Status.IN_PROGRESS, enum_class=tasks.models.Status,
```

Reply:
It sets to tasks.models.Status..BACKLOG , is it really matter to change it to tasks.models.Priority.IN_PROGRESS ?

Reply:
same ^^

Reply:
class Status(ChoiceEnum):
    BACKLOG = 'Backlog',
    IN_PROGRESS = 'InProgress',
    DONE = 'Done',
 BACKLOG is status 0, no?

Reply:
Yes backlog is 0 and thats should be the default

Reply:
enumchoicefield library generate automatically the enum accordingly to what decided in the models aka 1 is backlog.
Its not like most enum when the first is 0. That's shouldn't be a concern because the migrate files generate automatically.

Reply:
it seems that BACKLOG is '1', cause the 'makemigrations' does it automatically @mgold1234